Assets/Scripts/Entity/Player/HealthPostProcessing.cs

```diff
@@ -0,0 +1,82 @@
+﻿using UnityEngine;
+using UnityEngine.Rendering.PostProcessing;
+using Utils;
+
+namespace Entity.Player
+{
+    public class HealthPostProcessing : MonoBehaviour
+    {
+        [Tooltip("The intensity curve that scales the red screen effect depending on the player's health.")]
+        [SerializeField]
+        private AnimationCurve intensityCurve;
+
+        [Tooltip("Maximum frequency of vignette pulsing.")]
+        [SerializeField]
+        private float maxVignettePulseFrequency;
+
+        [Tooltip("Intensity of the vignette pulse.")]
+        [SerializeField]
+        [Range(0, 1)]
+        private float vignettePulseIntensity;
+
+        [Tooltip("The minimum sanity the player can have in order to display the red screen. If sanity goes below, red screen is not shown, but sanity vignette takes over.")]
+        [SerializeField]
+        [Range(0, 100)]
+        private float sanityCap;
```

Comment:
Is the transition instantly?

Reply:
Yep, it is. We could also decrease the intensity over time based on the sanity... it's not much, but more to stuff to tweak 😅 Right now the red screen effect disappears when the black one from the sanity starts.